Common/MVKCommonEnvironment.h

```diff
@@ -75,6 +75,9 @@ extern "C" {
 /** Directive to identify public symbols. */
 #define MVK_PUBLIC_SYMBOL        __attribute__((visibility("default")))
 
+/** Directive to make a public alias of another symbol. */
+#define MVK_PUBLIC_ALIAS(a, t)   asm(".globl _" #a "; _" #a " = _" #t "\n")
```

Comment:
Hmmm...no way to alias C function name symbols without invoking `asm()`?

All function aliasing references I can find are for C++, which I assume is why you couldn't use that? And the `__attribute__((alias))` seems to be available...but it seem to require the declaration to include function params.

I'm amazed you uncovered this technique. Where did you find it?